src/lando/headless_api/tests/test_automation_api.py

```diff
@@ -734,91 +736,128 @@ def test_automation_job_create_commit_success(
     assert len(job.landed_commit_id) == 40, "Landed commit ID should be a 40-char SHA."
 
 
+BAD_ACTION_TYPES = [
```

Comment:
nit: Can we use something more specific than `BAD_` as a prefix? Maybe `INVALID_` or `DISALLOWED_` or something else more specific.